vpc/vpc-meta.template

```diff
@@ -226,6 +226,34 @@
         ]
       }
     },
+    "NACLAdminGroup": {
+      "Type": "AWS::IAM::Group",
+      "Properties": {
+        "NACLAdminPolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+        {
+            "Sid": "Stmt1445360254279",
+            "Action": [
+                "ec2:CreateNetworkAcl",
+                "ec2:CreateNetworkAclEntry",
+                "ec2:DeleteNetworkAcl",
+                "ec2:DeleteNetworkAclEntry",
+                "ec2:DescribeNetworkAcls",
+                "ec2:ReplaceNetworkAclAssociation",
```

Comment:
If all that's needed is the ability to add entries to an existing ACL, I don't see how that privilege is needed.